BeVolt/Tests/Test_BSP_Strobelight.c

```diff
@@ -0,0 +1,22 @@
+#include "../BSP/Inc/BSP_Strobelight.h"
+#include <stdio.h>
+
+static const char* file = "BSP/Simulator/DataGeneration/Data/Strobe.csv";
+char a;
+int  main() {
+BSP_Strobe_Init();
+BSP_Strobe_On();
+FILE* fp = fopen(file, "r");
+a = getc(fp);
+fclose(fp);
```

Comment:
The reading of the strobelight status should be done on the Python side. This way you can test that both sides are working properly together.

Reply:
So what's the point of the C test file then?

Reply:
The C file should set the different states of the strobe light. Just as if the actual microcontroller code would do in certain situations. 

The python file would then act as the physical hardware, displaying an output for us to visually see.